cobalt/build/testing/targets/android-arm/test_targets.json

```diff
@@ -25,5 +25,31 @@
     "third_party/zlib:zlib_unittests",
     "url:url_perftests",
     "url:url_unittests"
+  ],
+  "executables": [
+    "out/android-arm_devel/apks/base_perftests-debug.apk",
```

Comment:
Like I said in chat, should be `out/android-arm_devel/base_perftests_apk/base_perftests-debug.apk` (and probably similar for all of them)